gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcQualityRuleRepository.java

```diff
@@ -53,4 +60,22 @@ protected JdbcObjectMapper<QualityRule> buildOrm() {
     protected String getId(QualityRule item) {
         return item.getId();
     }
+
+    @Override
+    public List<QualityRule> findByReference(QualityRuleReferenceType referenceType, String referenceId) throws TechnicalException {
+        LOGGER.debug("JdbcQualityRuleRepository.findByReference({}, {})", referenceType, referenceId);
+        try {
+            return jdbcTemplate.query(
+                getOrm().getSelectAllSql() + " where reference_type = ? and reference_id = ?",
```

Comment:
maybe I miss it but it would probably be interesting to add index on those fields ?